renderers/src/main/java/com/pedrogomez/renderers/RVRendererAdapter.java

```diff
@@ -40,8 +40,8 @@
  */
 public class RVRendererAdapter<T> extends RecyclerView.Adapter<RendererViewHolder> {
 
-  private final RendererBuilder<T> rendererBuilder;
-  private AdapteeCollection<T> collection;
+  protected final RendererBuilder<T> rendererBuilder;
+  protected AdapteeCollection<T> collection;
```

Comment:
Hi @darienalvarez thanks for your PR. Sorry for the delay reviewing this PR, I've been quite busy. In order to improve the class extensibility, I'd go for a protected getter method instead of just changing the attribute visibility. If we apply this change it could be easier in the future to protect the internal behavior of the library if needed. Could you apply this change?